### DISCUSSION TITLE
Allow easy Nuxi extendability

### DISCUSSION THREAD

**Discussion by rohrig** [nuxt/nuxt#17564] [Ideas]
It would be cool and convenient to be able to create commands quickly and easily via nuxi. Something similar to artisan make:command in Laravel. I would be open to helping out with the implementation. Please let me know if anyone else would find this useful. 

Also, command authors could summit commands to become part of the core set of nuxi commands. Of course, they would have to be  kick-ass commands that make the average Nuxt Dev's life easier.

**Comment by pi0** [comment]
Hi @rohrig, and thanks for the suggestion. Actually we do plan to do this! 

- Same as Nuxt 2, if you have a binary called `nuxt-xyz`, `nuxi xyz` alias is working
- Creating a new UnJS package that exposes common utilities to make a CLI (however you are always free to choose your own implementation or lib!)
